src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Performance/BasicPreferHashDataOverComputeHash.Fixer.vb

```diff
@@ -0,0 +1,31 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Diagnostics.CodeAnalysis
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.CodeFixes
+Imports Microsoft.CodeAnalysis.Editing
+Imports Microsoft.CodeAnalysis.Formatting
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Microsoft.NetCore.Analyzers.Performance
+
+Namespace Microsoft.NetCore.VisualBasic.Analyzers.Performance
+    <ExportCodeFixProvider(LanguageNames.VisualBasic)>
+    Public Class BasicPreferHashDataOverComputeHashFixer : Inherits PreferHashDataOverComputeHashFixer
+        Protected Overrides Function TryGetCodeFixer(computeHashNode As SyntaxNode, bufferArgNode As SyntaxNode, nodeToRemove As SyntaxNode, <NotNullWhen(True)> ByRef codeFixer As ApplyCodeFixAction) As Boolean
+            Dim usingStatement = TryCast(nodeToRemove, UsingStatementSyntax)
+            Dim usingBlock = TryCast(usingStatement.Parent, UsingBlockSyntax)
```

Comment:
Can the first cast fail? This will cause a NRE on the second statement.

Reply:
Added check